examples/cv/mnist_lenet5_image_classification_pure_lightning.py

```diff
@@ -48,6 +48,28 @@ def main(cfg: DictConfig):
     # The "model" - with dataloader/dataset inside of it.
     lenet5 = MNISTLeNet5(cfg.model)
 
+    # Setup train data loader and optimizer
+    lenet5.setup_training_data()
+
+    # Setup optimizer and scheduler
+    if 'sched' in cfg.model.optim:
```

Comment:
it seems to me that we'll be moving this logic inside models' setup_optimization() method. (in different PR).